src/main/java/com/google/sps/data/BackgroundTaskManagerInterface.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.data;
+import java.util.Optional;
+import java.util.List;
+
+// Interface for the BackgroundTaskManager class.
```

Comment:
Improper class javadoc format. Please fix. 

nit: You should not define an interface by its concrete class(es) like you've done. An interface is a standalone contract that describes the type of behavior/work that classes who subclass it will have/do. So technically, your comment should say something like:
"An Interface that describes the different types of background work that can be performed on VMN datastore entities."